.golangci.yml

```diff
@@ -0,0 +1,44 @@
+linters-settings:
+  dupl:
+    threshold: 100
+  goconst:
+    min-len: 2
+    min-occurrences: 2
+  gocyclo:
+    min-complexity: 20
+  gofmt:
+    simplify: true
+  golint:
+    # minimal confidence for issues, default is 0.8
+    min-confidence: 0
+  govet:
+    check-shadowing: true
+  misspell:
+    locale: US
+  maligned:
+    suggest-new: true
+  
+linters:
+  enable:
```

Comment:
I really want to add https://github.com/walle/lll to this list, but it's just to much work to fix all the long line for now.

Reply:
It looks good. I think that long line looks ugly and may cause bugs because of the difficulty of understanding.

However, if we limit long line, are there any guidelines or rules to follow?
For example, how should we handle the following line?

```go
func (p *MatterpollPlugin) handleAddOptionDialogRequest(vars map[string]string, request *model.PostActionIntegrationRequest) (*i18n.Message, *model.Post, error) {
```

Reply:
I was wondering exactly the same :wink:

Reply:
Would you be fine if we most this discussion into a new issue/PR?

Reply:
Sure!